recipes/jwasm/all/conanfile.py

```diff
@@ -34,6 +35,10 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        # INFO: Upstream recommends using C99 to avoid errors with C23
+        # https://github.com/Baron-von-Riedesel/JWasm/issues/39#issuecomment-2823163402
+        if not self.settings.get_safe("compiler.cstd") and self.settings.compiler in ["gcc", "clang", "apple-clang"]:
```

Comment:
One comment about this. I saw that this flag is already added upstream: https://github.com/Baron-von-Riedesel/JWasm/commit/4f4d260186fd6b222fd10c1c894557f65c741956#diff-ec37181a8b48dd8101d16540702f2d047a7fc726ffe9ecf5e3e59bb7419e1ce7L22, but only for GCC compiler.

Reply:
Good catch! It was added recently, hope we can have a new release with a proper fix soon.